tests/db/test_bigchain_api.py

```diff
@@ -257,9 +257,38 @@ def test_read_transaction_invalid_block(self, b, user_vk, user_sk):
         response = b.get_transaction(tx.id)
 
         # should be None, because invalid blocks are ignored
+        # and a copy of the tx is not in the backlog
         assert response is None
 
     @pytest.mark.usefixtures('inputs')
+    def test_read_transaction_invalid_block_and_backlog(self, b, user_vk, user_sk):
+        from bigchaindb.models import Transaction
+
+        input_tx = b.get_owned_ids(user_vk).pop()
+        input_tx = b.get_transaction(input_tx.txid)
+        inputs = input_tx.to_inputs()
+        tx = Transaction.transfer(inputs, [user_vk], input_tx.asset)
+        tx = tx.sign([user_sk])
```

Comment:
Yeah, but I'm not sure how to go about that. I based the two "new" tests on the one original test, so at least this is no worse than before.